### PR TITLE
[221796][Task][User]: Code & Document: Create - Retrieve - Update - List - Delete

### PR DESCRIPTION
**Pull request by tungmaiphuoc**
https://dev.framgia.com/issues/222026
https://dev.framgia.com/issues/222028
https://dev.framgia.com/issues/222029
https://dev.framgia.com/issues/222031
https://dev.framgia.com/issues/222032
https://dev.framgia.com/issues/222034
https://dev.framgia.com/issues/222035
https://dev.framgia.com/issues/222037
https://dev.framgia.com/issues/222038
https://dev.framgia.com/issues/222040

### CODE REVIEW

File: docs/USER.md [hunk 1]
```diff
@@ -0,0 +1,322 @@
...
+log.Printf("%s\n", response)
+```
+
+Example Response
+
+```golang
+{  
+  "data":[  
+    {  
+      "id":"7d5eb2ed-0be5-4a20-a84a-f2e6f959aa84",
```
Comment (titbk) [inline docs/USER.md]: List bỏ mẫu vài cái thôi ạ 
anh add limit cho UserListParams = 2 thôi

File: docs/USER.md [hunk 2]
```diff
@@ -0,0 +1,322 @@
...
+response, _ := user.Delete(params)
+log.Printf("%s\n", response)
+```
+
+Example Response
+
+```golang
+{
+    "data": {
+        "result": true
```
Comment (titbk) [inline docs/USER.md]: Phần delete user trả về khác các trường hợp khác vậy ak anh, không phải là một id trong data ak

File: user/client.go [hunk 3]
```diff
@@ -0,0 +1,101 @@
...
+
+// Get Backend Client
+func getC() Client {
+	b := uiza.GetBackend(uiza.APIBackend)
+	b.SetClientType(uiza.UserClientType)
+	return Client{b, uiza.Key}
+}
+
+func Retrieve(params *uiza.UserIDParams) (*uiza.UserData, error) {
+
```
Comment (titbk) [inline user/client.go]: bỏ cách hàng không cần thiết lun anh

File: docs/USER.md (hunk 2 above)
Comment (tungmaiphuoc) [reply docs/USER.md]: ko e, a đã confirm trên postman, kết quả trả về là "result": true

File: docs/USER.md (hunk 1 above)
Comment (tungmaiphuoc) [reply docs/USER.md]: Đã fix

File: user/client.go (hunk 3 above)
Comment (tungmaiphuoc) [reply user/client.go]: Đã fix

**Pull request merged**